spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
```

Comment:
You should have a context that describes what happens when no user is signed in.